clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/TransferReadGroupV1QueryOutput.scala

```diff
@@ -3,15 +3,53 @@ package org.broadinstitute.clio.transfer.model
 import java.time.OffsetDateTime
 
 case class TransferReadGroupV1QueryOutput(
```

Comment:
In some cases I see `V1` in names and in other cases I don't, e.g. `TransferReadGroupV1QueryOutput` vs `querySchema`. Is there a reason for this difference?